### PR TITLE
New topic: tealang

### PR DESCRIPTION
**Pull request by zrzjohn**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> A new programming language.

---------------------------------------------------------------------

### CODE REVIEW

File: topics/tea/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+aliases: tealang
+created_by: Benny
+display_name: Tea
+github_url: https://github.com/tealang
+related: language
+released: '2019'
+short_description: Tea is a humanized programming language.
+topic: tea
+url: https://tealang.github.io/
```
Comment (kenyonj) [inline topics/tea/index.md:10]:
```suggestion
created_by: Benny
display_name: Tea
github_url: https://github.com/tealang
related: language
released: '2019'
short_description: Tea is a humanized programming language.
topic: tealang
url: https://tealang.github.io/
```

**Review by kenyonj — CHANGES_REQUESTED** [review]
Unfortunately there seems to be a more widely used acronym definition for TEA currently: "The Elm Architecture". Please update this topic name to be more specific to tealang, perhaps the suggestion I've left for `tealang`

**Comment by zrzjohn** [comment]
@kenyonj It has been changed to tealang.

**Comment by kenyonj** [comment]
Sorry for the back and forth here, but now that we've changed to `tealang` as the topic, it's apparent that this does not meet our requirements for new topics. See here: https://github.com/github/explore/blob/master/CONTRIBUTING.md

> We are likely to consider suggestions to curate a topic or collection that is valuable to GitHub's community. Valuable topics, for example, include those that are already widely used by repositories and could benefit from the addition of important information. When suggesting content, please consider how to make your contribution broadly useful and relevant to others, rather than serving a specific use case.

**Pull request closed without merging**